PiwikTracker/PiwikTracker.swift

```diff
@@ -22,6 +22,8 @@ final public class PiwikTracker: NSObject {
     private var queue: Queue
     internal let siteId: String
     
+    public var logger: Logger? = DefaultLogger(minLevel: .warning)
+    
     internal static var _sharedInstance: PiwikTracker?
```

Comment:
I personally think `_` prefixing is a thing of the past but accept if someone else thinks otherwise :)